### PR TITLE
V3 upgrade

### PR DESCRIPTION
**Pull request by admataz**
Try again - tests now passing in CI for all target node versions

- provide a valid gql schema to test registrations (default of empty fields throws)
- wrap invalid schema registrations for node v14 so error is caught

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,28 +28,31 @@
...
+    "fastify-websocket": "^2.0.0",
     "graphql": "^15.0.0",
     "graphql-jit": "^0.4.0",
     "http-errors": "^1.7.3",
     "mqemitter": "^4.0.0",
     "p-map": "^4.0.0",
     "readable-stream": "^3.5.0",
     "single-user-cache": "^0.3.0",
-    "tiny-lru": "^7.0.2",
-    "ws": "^7.2.1"
```
Comment (mcollina) [inline package.json:53]: why was `ws` moved to a devDependency?

File: .dependabot/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+version: 1
+update_conigs:
```
Comment (mcollina) [inline .dependabot/config.yml:2]: this has a typo... `configs`

File: package.json (hunk 1 above)
Comment (admataz) [reply package.json:53]: must be a bad merge on my part when resolving conflicts. fixed

File: package.json [hunk 3]
```diff
@@ -28,20 +28,20 @@
...
-    "fastify": "^2.12.0",
+    "fastify": "^3.0.0",
     "graphql-tools": "^6.0.8",
     "pre-commit": "^1.2.2",
     "proxyquire": "^2.1.3",
     "snazzy": "^8.0.0",
     "standard": "^14.0.0",
     "tap": "^14.10.6",
-    "typescript": "^3.7.5"
+    "tsd": "^0.11.0"
```
Comment (mcollina) [inline package.json:38]: Can you up[date this to `^0.13.1`?

**Pull request merged**